src/Tribe/Tickets.php

```diff
@@ -4368,14 +4369,20 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 			$meta = tribe( 'tickets-plus.meta' );
 
 			$cart_has_meta = true;
+			$cart_has_iac  = true;
 
-			// If the method exists (latest ET+ version), run it.
+			// If the method exists in ET+, run it.
 			if ( method_exists( $meta, 'cart_has_meta' ) ) {
 				$cart_has_meta = $meta->cart_has_meta( $tickets_in_cart );
 			}
 
-			// There are no meta fields on the cart tickets.
-			if ( ! $cart_has_meta ) {
+			// If the method exists in ET+, run it.
+			if ( method_exists( $meta, 'cart_has_iac' ) ) {
+				$cart_has_iac = $meta->cart_has_iac( $tickets_in_cart );
+			}
```

Comment:
Can we do this through a filter ? I would not introduce the $cart_has_iac variable here at all. I would apply_filters here whether should we bail or not and hook into ETP.

Honestly this whole method should be in ETP instead, if you could move would be awesome. Othewise let's just not continue adding ETP logic inside ET :D